docs/about.md

```diff
@@ -0,0 +1,6 @@
+---
+title: About Us
+path: /about/
+---
+
+# About us
```

Comment:
Are we filling this up or not? Just a comment

Reply:
Yea but this is mostly just setting it up. Nothing else really has content beyond a few migrated docs from different repos. It all needs work but it's gonna have to wait until this merges imo.